client/src/components/editor/RichTextEditor.css

```
@@ -45,3 +45,20 @@
 .editor-link-chooser {
   z-index: 1300;
 }
+
+/* 
+ * Rich text editor style overrides.
+ * when making a change, change rich-text-readonly class accordingly.
+*/
+
+.editable h1 {
+  font-size: 1.75rem;
+}
+
+.editable h2 {
+  font-size: 1.5rem;
+}
+
+.editable h3 {
+  font-size: 1.25rem;
+}
```

I propose to take even smaller fonts fot the headings:
```suggestion
.editable h1 {
  font-size: 1.5rem;
}

.editable h2 {
  font-size: 1.25rem;
}

.editable h3 {
  font-size: 1.0rem;
}
```